src/creature/creature.cpp

```diff
@@ -38,6 +39,7 @@ void Creature::PrintStats()
 	std::cout << "Wisdom: " << m_ability_scores[4] << std::endl;
 	std::cout << "Charisma: " << m_ability_scores[5] << std::endl;
 	std::cout << "Creature type: " << m_creature_type << std::endl;
+	std::cout << "Display name: " << m_display_name << std::endl;
```

Comment:
Nice catch! Although, I would propose to change line 31, "Your stats are:" with "Stats for " `m_display_name` ":"
Then, we don't need that extra line here.